corehq/apps/users/templates/users/filter_and_download.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'users/base_template.html' %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+
+{% block js-inline %}{{ block.super }}
+<script>
```

Comment:
I think that this should be in a separate javascript file and you should use initial_page_data to pass any data to it.